cmd/ruler/main.go

```diff
@@ -59,28 +58,29 @@ func main() {
 	util.CheckFatal("", err)
 	defer chunkStore.Stop()
 
-	r, err := ring.New(ringConfig)
+	r, err := ring.New(lifecyclerConfig.RingConfig)
```

Comment:
I'd like us to have two ringConfigs here - one for the ingesters and one for the rulers.  This will allow us to store them in different consuls in the future.  They should register their flags with different prefixes.